Mathlib/Algebra/Order/Field/Defs.lean

```diff
@@ -35,3 +35,8 @@ class LinearOrderedField (α : Type*) extends LinearOrderedCommRing α, Field α
 instance (priority := 100) LinearOrderedField.toLinearOrderedSemifield [LinearOrderedField α] :
     LinearOrderedSemifield α :=
   { LinearOrderedRing.toLinearOrderedSemiring, ‹LinearOrderedField α› with }
+
+variable [LinearOrderedSemifield α] {a : α}
+
+lemma mul_inv_le_one : a * a⁻¹ ≤ 1 := by obtain rfl | ha := eq_or_ne a 0 <;> simp [*]
+lemma inv_mul_le_one : a⁻¹ * a ≤ 1 := by obtain rfl | ha := eq_or_ne a 0 <;> simp [*]
```

Comment:
Do you think it makes sense to have the `cancel` versions we have for equalities here too?

A docstring for each of these referencing the junk value and the other lemma would be good too.

Reply:
Hmm, that's a good idea. Will add